src/webview/serverless-function/app/home.tsx

```diff
@@ -31,28 +58,67 @@ export class ServerlessFunction extends React.Component<DefaultProps, {
         });
     }
 
+    handleInvokeSubmit = (name: string, instance: string, id: string, path: string, contentType: string, format: string, source: string,
+        type: string, data: string, file: string, enableUrl: boolean, invokeURL: string): void => {
+        VSCodeMessage.postMessage({
+            action: 'invokeFunction',
+            name: name,
+            instance: instance,
+            id: id,
+            path: path,
+            contentType: contentType,
+            format: format,
+            source: source,
+            type: type,
+            data: data,
+            file: file,
+            enableURL: enableUrl,
+            invokeURL: invokeURL
+        });
+    }
+
     render(): React.ReactNode {
         return (
-            <div className='mainContainer margin'>
-                <div className='title'>
-                    <Typography variant='h5'>OpenShift Serverless Functions</Typography>
+            this.state.invoke ?
+                <div className='mainContainer margin'>
+                    <div className='title'>
+                        <Typography variant='h5'>Invoke Function</Typography>
+                    </div>
+                    <Container maxWidth='md' sx={{
+                        border: '1px groove var(--vscode-activityBar-activeBorder)',
+                        borderRadius: '1rem', margin: 'auto', backgroundColor: '#101418',
+                        color: '#99CCF3'
+                    }}>
+                        <Box
+                            display='flex'
+                            flexDirection={'column'}
+                        >
+                            <InvokeFunction instance={this.state.instance} uri={this.state.folderURI} name={this.state.name}
+                                invokeURL={this.state.invokeURL} id={this.state.id} onInvokeSubmit={this.handleInvokeSubmit} />
+                        </Box>
+                    </Container>
```

Comment:
Move the code surrounding the invoke component and the create function component into their respective components. eg.:

```tsx
this.state.invoke ?
<InvokeFunction />
: <CreateFunction />
```

Then put the title and container into the `InvokeFunction` and `CreateFunction`